### ISSUE TITLE
Syntax Highlighting for Atom

### ISSUE THREAD

**Issue by pi0neerpat** [earthly/earthly#1886]
Request to create a syntax highlight package for Atom IDE https://atom.io/

Related to #1203 #359

![image](https://user-images.githubusercontent.com/35622595/167942538-f8e792a1-8513-4784-ad8f-dbe15f65d644.png)

**Comment by wingyplus** [comment]
If we want to support on Atom, I think we should explore on tree-sitter-grammar for Earthly. It can be apply more text editor that support tree-sitter by default (neovim, emacs with emacs-tree-sitter package, helix, etc.).

What do you think?

**Comment by pi0neerpat** [comment]
I'm not familiar with the specifics, but if there's a standard language for syntax tooling, then thats definitely a good idea!

**Comment by jmgilman** [comment]
I agree focusing on tree-sitter is the optimal path here. It's seeing widespread adoption and is likely to pay off even more in the future.
